assignments/function-conversion.js

```diff
@@ -10,6 +10,8 @@
 //   return param1 + param2;
 // };
 // add(1,2);
+let add = (x, y) => {return x + y};
```

Comment:
Good! When you have it on one line like this, you can also write it as `let add = (x, y) => x + y;`. When on one line like this you can skip the curly braces `{}` and the `return` keyword, as it does that implicitly.
When you use the arrow function on multi lines however you _do_ need the curly braces and return keyword.